AssertionError: 'true' not found in: tests.upgrades.test_repository.Scenario_repository_upstream_authorization_check testMethod=test_pre_repository_scenario_upstream_authorization

E       AssertionError: 'true' not found in '/usr/share/foreman/lib/foreman.rb:8: warning: already initialized constant Foreman::UUID_REGEXP\r\n/usr/share/foreman/lib/foreman.rb:8: warning: previous definition of UUID_REGEXP was here\r\n/usr/share/foreman/lib/core_extensions.rb:182: warning: already initialized constant ActiveSupport::MessageEncryptor::DEFAULT_CIPHER\r\n/opt/theforeman/tfm-ror51/root/usr/share/gems/gems/activesupport-5.1.6/lib/active_support/message_encryptor.rb:22: warning: previous definition of DEFAULT_CIPHER was here\r\n/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.7.0.42/app/models/katello/concerns/content_facet_host_extensions.rb:7: warning: already initialized constant Katello::Concerns::ContentFacetHostExtensions::ERRATA_STATUS_MAP\r\n/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.7.0.42/app/models/katello/concerns/content_facet_host_extensions.rb:7: warning: previous definition of ERRATA_STATUS_MAP was here\r\n/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.7.0.42/app/models/katello/concerns/content_facet_host_extensions.rb:14: warning: already initialized constant Katello::Concerns::ContentFacetHostExtensions::TRACE_STATUS_MAP\r\n/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.7.0.42/app/models/katello/concerns/content_facet_host_extensions.rb:14: warning: previous definition of TRACE_STATUS_MAP was here\r\n/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.7.0.42/app/models/katello/concerns/subscription_facet_host_extensions.rb:13: warning: already initialized constant Katello::Concerns::SubscriptionFacetHostExtensions::SUBSCRIPTION_STATUS_MAP\r\n/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.7.0.42/app/models/katello/concerns/subscription_facet_host_extensions.rb:13: warning: previous definition of SUBSCRIPTION_STATUS_MAP was here\r\nLoading production environment (Rails 5.1.6)\r\nFailed to load console gems, starting anyway\r\nSwitch to inspect mode.\r\nrepo = Katello::RootRepository.find_by_id(498); repo.upstream_username = "rTtest123"; repo.save!(validate: false)\r\nNameError: uninitialized constant Katello::RootRepository\r\n\tfrom foreman_hooks (0.3.14.1) lib/foreman_hooks/as_dependencies_hook.rb:4:in `load_missing_constant\'\r\n\tfrom (irb):1\r\n\tfrom lib/tasks/console.rake:5:in `block in <top (required)>\''

--- a/tests/upgrades/test_repository.py
+++ b/tests/upgrades/test_repository.py
@@ -56,7 +56,7 @@ class Scenario_repository_upstream_authorization_check(APITestCase):
 
         org = entities.Organization().create()
         custom_repo = create_sync_custom_repo(org_id=org.id)
-        rake_repo = 'repo = Katello::RootRepository.find_by_id({0})'.format(custom_repo)
+        rake_repo = 'repo = Katello::Repository.find_by_id({0})'.format(custom_repo)
         rake_username = '; repo.upstream_username = "{0}"'.format(self.upstream_username)
         rake_repo_save = '; repo.save!(validate: false)'
         result = run("echo '{0}{1}{2}'|foreman-rake console".format(rake_repo, rake_username,

